CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 - Handle errors when sample sheet not provided or doesn't have a schema ([#56](https://github.com/nextflow-io/nf-validation/pull/56))
 - Silently ignore samplesheet fields that are not defined in samplesheet schema ([#59](https://github.com/nextflow-io/nf-validation/pull/59))
 - Correctly handle double-quoted fields containing commas in csv files by `.fromSamplesheet()` [#63](https://github.com/nextflow-io/nf-validation/pull/63))
+- Assume that a value with no default in the schema will have a default of 'false' ([#66](https://github.com/nextflow-io/nf-validation/pull/66))
```

Comment:
I'm not sure that this is accurate?

Sounds like we are setting a default, but it's just for the summary, right?

```suggestion
- Do not return parameter in summary if it has no default in the schema and is set to 'false' ([#66](https://github.com/nextflow-io/nf-validation/pull/66))
```

Reply:
This sounds better, thanks for the suggestion!